Predictores/predictor_ideal.py

```diff
@@ -140,8 +150,8 @@ def predict(self, Q: np.ndarray) -> np.ndarray:
         if self.prediction_denorm:
             return self.output[..., :-1]
         else:
-            columns = self.prediction_features_names + ['Q']
-            return normalize_numpy_array(self.output, columns, self.normalization_info)[:, :-1]
+            # columns = self.prediction_features_names + ['Q']
+            # return normalize_numpy_array(self.output, columns, self.normalization_info)[:, :-1]
```

Comment:
This branch is never accessed since I set prediction_denorm to true.
Do we want to keep it? Its functionality might not work for the case where batches of states are used. I couldn't test it because I had no normalization info file at hand.

Reply:
Fixed